### PR TITLE
Convert yet more `import *` to `import =`

### PR DESCRIPTION
**Pull request by ghost**
Sequel to #23461

### CODE REVIEW

**Review by westy92 — APPROVED** [review]

**Comment by geppy** [comment]
[reasoning behind the changes](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23461#issuecomment-363519096) (re: [Microsoft/TypeScript#21535](https://github.com/Microsoft/TypeScript/issues/21535))

**Review by geppy — APPROVED** [review]
Approving the PouchDB changes, at least.

**Review by charrondev — APPROVED** [review]

**Review by sgmccli — APPROVED** [review]
Approving parent-package-json

**Review by shiftkey — APPROVED** [review]

**Review by bradzacher — APPROVED** [review]

**Review by Sayan751 — APPROVED** [review]

**Comment by Archcry** [comment]
Approving changes.

**Review by sanderkoenders — APPROVED** [review]

**Review by andraaspar — APPROVED** [review]

**Review by taoqf — APPROVED** [review]

**Review by j-f1 — APPROVED** [review]

**Review by DenisCarriere — APPROVED** [review]

**Review by jacobbaskin — APPROVED** [review]

**Review by spacejack — APPROVED** [review]

**Review by dead-claudia — APPROVED** [review]

File: types/mithril/test/test-api.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // Typescript adaptation of mithril's test suite.
 // Not intended to be run; only to compile & check types.
 
-import * as m from 'mithril';
-import * as stream from 'mithril/stream';
+import m = require('mithril');
+import stream = require('mithril/stream');
```
Comment (isiahmeadows) [inline types/mithril/test/test-api.ts]: @spacejack We really *should* convert Mithril's value-level exports to *just* be default exports, so we remain ES6-compatible. Yes, it's breaking, but we're going to have to do it regardless.

**Review by jcchalte — APPROVED** [review]

**Pull request merged**